mongodb/blank/index.js

```diff
@@ -0,0 +1,96 @@
+const { MongoClient } = require('mongodb');
```

Comment:
Is there a ESM version of this? or require the default method?

Reply:
I'll switch to importing the default export.